nsxt/resource_nsxt_policy_group.go

```diff
@@ -117,6 +124,27 @@ func getMACAddressExpressionSchema() *schema.Resource {
 	}
 }
 
+func getExternalIDExpressionSchema() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"external_ids": {
+				Type:        schema.TypeSet,
+				Required:    true,
+				Description: "List of external ids",
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"member_type": {
+				Type:         schema.TypeString,
+				Required:     true,
```

Comment:
I think we can make this optional with default being `VirtualMachine`

Reply:
Sure. Let me change it. Thanks!